core/src/main/java/org/elasticsearch/action/get/GetResponse.java

```diff
@@ -168,4 +169,9 @@ public void writeTo(StreamOutput out) throws IOException {
         super.writeTo(out);
         getResult.writeTo(out);
     }
+
+	@Override
```

Comment:
Oh no! The build failed because of tabs!

You can catch these sorts of things with `gradle core:precommit` which is much faster than `gradle core:check` because it doesn't run all the tests. I'll run the tests before pushing but for something like this it ought to be safe for you not to run them.